rir/src/interpreter/cache.h

```diff
@@ -1,56 +1,104 @@
 #ifndef RIR_INTERPRETER_CACHE_H
 #define RIR_INTERPRETER_CACHE_H
 
+#include "R/r.h"
 #include "instance.h"
 
 namespace rir {
 
-// must be a power of 2 for modulus using & CACHE_MASK to work
-#define MAX_CACHE_SIZE 128
-#define CACHE_MASK (MAX_CACHE_SIZE - 1)
+// makes max cache size fit into 64bytes (with size)
```

Comment:
the cache datastructure now also contains the size. because reading out the size from c was fishy (think about the case where c changes but the cache not...)

Reply:
Yes, that I got it. But what max cache size fit into 64 bytes means in this context?

Reply:
64bytes = L1 cacheline